test/steps/1_given.js

```diff
@@ -16,17 +16,55 @@
 import fs from 'fs';
 import readline from 'readline';
 import lisk from 'lisk-js';
+import Vorpal from 'vorpal';
 import defaultConfig from '../../defaultConfig.json';
 import cryptoInstance from '../../src/utils/cryptoModule';
 import * as fsUtils from '../../src/utils/fs';
 import liskInstance from '../../src/utils/liskInstance';
+import * as mnemonicInstance from '../../src/utils/mnemonic';
 import queryInstance from '../../src/utils/query';
 import {
 	getFirstQuotedString,
 	getQuotedStrings,
 	createFakeInterface,
 	createStreamStub,
 } from './utils';
+import createAccountCommand from '../../src/commands/createAccount';
+
+export function aValidMnemonicPassphrase() {
+	this.test.ctx.mnemonicPassphrase = getFirstQuotedString(this.test.parent.title);
+}
+
+export function anInvalidMnemonicPassphrase() {
+	this.test.ctx.mnemonicPassphrase = getFirstQuotedString(this.test.parent.title);
+}
+
+export function thePassphraseIsGeneratedByTheCreateMnemonicPassphraseFunction() {
+	this.test.ctx.MnemonicStub = sandbox.stub(mnemonicInstance, 'createMnemonicPassphrase').returns(this.test.ctx.passphrase);
```

Comment:
We shouldn't need to store this in the test context as we can just access it later on directly if we need it.